docs/profiling.md

```diff
@@ -27,3 +28,23 @@ startProfiling({
 
 startTracing({ ... });
 ```
+
+
+### Memory profiling
+
+Memory profiling is disabled by default, it can be enabled via the `memoryProfilingEnabled` flag.
```

Comment:
Suggestion: Memory profiling is disabled by default. You can enable it via the `memoryProfilingEnabled` flag.

Reply:
Thanks! Perhaps you already know, but GitHub lets you add suggestions directly to the comment via a `suggest` code block, so the author can press a button to bring the change in, for example:

```suggestion
Memory profiling is disabled by default. You can enable it via the `memoryProfilingEnabled` flag.
```